### PR TITLE
Changes in SuperAdminSettingScreen

### PR DESCRIPTION
**Pull request by JitendraSaini1995**
In SuperAdminSettingScreen following changes are applied:
register button & set test employee button removed,siteName textfield added and textfields made uneditable.

### CODE REVIEW

**Comment by JitendraSaini1995** [comment]
https://drive.google.com/file/d/14DqIa3N-xLndIugREpOUXp7_Lbkp_0Ev/view?usp=sharing.
Screenshot 2024-02-20 142737

File: app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt [hunk 1]
```diff
@@ -177,26 +196,12 @@ fun SuperAdminSettingScreen(
                     style = MaterialTheme.typography.titleSmall
                 )
             }
-            Row(
-                horizontalArrangement = Arrangement.Center,
-                verticalAlignment = Alignment.CenterVertically,
+            Button(
+                onClick = { /*TODO*/ },
+                shape = RoundedCornerShape(MaterialTheme.dimens.superAdminSettingScreenButtonsCornerShapeSize),
+                colors = ButtonDefaults.buttonColors(containerColor = Color(0xFF85D32C)),
```
Comment (mohit2988) [inline app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt:202]: Make the Color class and name it OnTimeColors and add this Color `Color(0xFF85D32C)` in that file.
exa. val colorXYZ = Color(0xFF85D32C)
ButtonDefaults.buttonColors(containerColor = OnTimeColors.colorXYZ)

Comment (mohit2988) [reply app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt:202]: And find all the colors which are in this file and update as suggested and use the same in future

File: app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt [hunk 2]
```diff
@@ -363,27 +370,31 @@ private fun SuperAdminSettingInfo(
                 InputField(
                     valueState = latLngState,
                     labelId = "",
-                    enabled = true,
+                    enabled = false,
```
Comment (mohit2988) [inline app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt:373]: Why are we making it false?
Your InputField composable may have default values which is common for everywhere
Like
InputField(enabled : Boolean =false)

File: app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt [hunk 3]
```diff
@@ -363,27 +370,31 @@ private fun SuperAdminSettingInfo(
...
-                        .size(
-                            width = MaterialTheme.dimens.superAdminSettingScreenRegisterButtonWidth,
-                            height = MaterialTheme.dimens.superAdminSettingScreenRegisterButtonHeight
-                        )
-                        .padding(MaterialTheme.dimens.superAdminSettingScreenColumnStartPadding)
-                ) {
-                    Text(text = stringResource(id = R.string.REGISTER))
-                }
+                Text(
+                    text = "Site Name",
```
Comment (mohit2988) [inline app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt:382]: Add this one in string file

**Review by mohit2988 — APPROVED** [review]
Approving the PR but pls incorporate with PR comment

File: app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt (hunk 3 above)
Comment (JitendraSaini1995) [reply app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt:382]: done!

File: app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt (hunk 2 above)
Comment (JitendraSaini1995) [reply app/src/main/java/com/allocate/ontime/presentation_logic/screens/super_admin/SuperAdminSettingScreen.kt:373]: done!

**Pull request merged**